### PR TITLE
minor grappler controller bugs

### PR DESCRIPTION
**Pull request by Benyuel**
* fixed (str.format bug when using HierarchicalController methods with verbose=True
* fixed binary str vs str mismatch bug in controller.py and hierarchical_controller.py

### CODE REVIEW

File: tensorflow/python/grappler/controller.py [hunk 1]
```diff
@@ -48,6 +48,8 @@ def __init__(self, item, cluster):
     # order. The order of this collection is deterministic.
     self.important_ops = []
     for name in important_op_names:
+      if isinstance(name, bytes):
```
Comment (benoitsteiner) [inline tensorflow/python/grappler/controller.py]: Is this still needed ? item.IdentifyImportantOps() should now return strings and not bytes.

File: tensorflow/python/grappler/hierarchical_controller.py [hunk 2]
```diff
@@ -464,6 +464,8 @@ def create_op_embeddings(self, verbose=False):
 
     self.type_dict = {}
     for op_type in self.cluster.ListAvailableOps():
+      if isinstance(op_type, bytes):
```
Comment (benoitsteiner) [inline tensorflow/python/grappler/hierarchical_controller.py]: Is this still needed ? cluster.ListAvailableOps() should now return strings and not bytes.

File: tensorflow/python/grappler/controller.py (hunk 1 above)
Comment (Benyuel) [reply tensorflow/python/grappler/controller.py]: I was looking at the latest nightly build and it appeared to be returning bytes.  I haven't built from source to see though.

File: tensorflow/python/grappler/hierarchical_controller.py (hunk 2 above)
Comment (Benyuel) [reply tensorflow/python/grappler/hierarchical_controller.py]: I was looking at the latest nightly build and it appeared to be returning bytes.  I haven't built from source to see though.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @benoitsteiner: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by tensorflowbutler** [comment]
It has been 43 days that this pull-request has stalled. Please create a new pull-request with the requested changes.

**Pull request closed without merging**